lab-michael/test/fs-readfile-test.js

```diff
@@ -0,0 +1,15 @@
+const readTheFile = require('../lib/fs-readfile.js');
+const assert = require('assert');
+
+describe('Readfile Module', function() {
+  describe('Testing method #readTheFile()', function() {
+    it('will match the first 8 bits', function(done) {
+      readTheFile(function(data) {
```

Comment:
Try separating these out into differing tests. You generally want to write an `it()` block for each assertion.